### PR TITLE
dev-python/argcomplete: version bump 1.6.0

### PR DESCRIPTION
**Pull request by anthonyryan1**
* EAPI=6
* Add pypy & pypy3 to PYTHON_COMPAT (tested upstream)
* Add ~arm to keywords (tested by me)
* Add new dependency on dev-python/pexpect for testing
* Add github to upstream metadata

### CODE REVIEW

File: dev-python/argcomplete/argcomplete-1.6.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} pypy pypy3 )
+
+inherit distutils-r1 bash-completion-r1
```
Comment (SoapGentoo) [inline dev-python/argcomplete/argcomplete-1.6.0.ebuild]: sort alphabetically

File: dev-python/argcomplete/argcomplete-1.6.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+HOMEPAGE="https://pypi.python.org/pypi/argcomplete"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="Apache-2.0"
+KEYWORDS="~amd64 ~x86 ~arm ~amd64-linux ~x86-linux ~x64-macos"
+IUSE="test"
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
+	test? ( dev-python/pexpect )"
```
Comment (SoapGentoo) [inline dev-python/argcomplete/argcomplete-1.6.0.ebuild]: pyexpect doesn't need `${PYTHON_USEDEP}`?

File: dev-python/argcomplete/argcomplete-1.6.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+python_prepare_all() {
+	# purge test folder to avoid file collisions
+	sed -e "s:find_packages():find_packages(exclude=['test','test.*']):" -i setup.py || die
+	distutils-r1_python_prepare_all
+}
+
+python_test() {
+	${EPYTHON} test/test.py || die
```
Comment (SoapGentoo) [inline dev-python/argcomplete/argcomplete-1.6.0.ebuild]: quote `${EPYTHON}`

File: dev-python/argcomplete/argcomplete-1.6.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+python_install_all() {
+	sed \
+		-e "/complete /d" \
+		-i argcomplete/bash_completion.d/python-argcomplete.sh || die
```
Comment (SoapGentoo) [inline dev-python/argcomplete/argcomplete-1.6.0.ebuild]: prefer doing the `sed` in the prepare pseudo-function

**Comment by kensington** [comment]
@gentoo/python

File: dev-python/argcomplete/argcomplete-1.6.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+LICENSE="Apache-2.0"
+KEYWORDS="~amd64 ~x86 ~arm ~amd64-linux ~x86-linux ~x64-macos"
+IUSE="test"
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
+	test? ( dev-python/pexpect[${PYTHON_USEDEP}] )"
+
+python_prepare_all() {
+	distutils-r1_python_prepare_all
+}
```
Comment (SoapGentoo) [inline dev-python/argcomplete/argcomplete-1.6.0.ebuild]: nix this, you're just calling the default pseudo-phase function

**Pull request closed without merging**